src/cmd/commit.ts

```diff
@@ -156,20 +154,18 @@ export const commit = async (): Promise<void> => {
     })
   }
   await preCommit()
-  if (values?.charts?.gitea?.enabled) await commitAndPush()
+  if (values?.apps?.gitea?.enabled) await commitAndPush()
```

Comment:
This is very confusing. I expect consistent behaviour regardless underining git repository.
The `otomi commit` command should not sometimes push changes and sometimes not.
I prefer that `otomi commit` just generate requred files validate-values but does not push it to the remote repo.